corehq/motech/repeaters/tasks.py

```diff
@@ -77,31 +80,53 @@ def delete_old_request_logs():
     queue=settings.CELERY_PERIODIC_QUEUE,
 )
 def check_repeaters():
+    # this creates a task for all partitions
+    # the Nth child task determines if a lock is available for the Nth partition
+    for current_partition in range(CHECK_REPEATERS_PARTITION_COUNT):
+        check_repeaters_in_partition.delay(current_partition, CHECK_REPEATERS_PARTITION_COUNT)
```

Comment:
nit: if this is a constant, you probably don't need to pass it here (unless you expect to call this function somewhere else with a different value)